control_panel_api/services.py

```diff
@@ -34,6 +39,44 @@ def app_slug(name):
     return re.sub(r'_+', '-', slugify(name))
 
 
+def app_create(app_slug):
+    """Creates the IAM role for the given app"""
+    _create_app_role(_app_role_name(app_slug))
+
+
+def _create_app_role(role_name):
+    """Creates the IAM role for the given app"""
+
+    # See: `sts:AssumeRole` required by kube2iam
+    # https://github.com/jtblin/kube2iam#iam-roles
+    assume_role_policy = {
```

Comment:
I think the policy document could be extracted out of the function. Maybe stored somehwere else or put in a function?  Some consistency how this is done with the bucket `get_policy_document()` would be good.

Reply:
This is a bit confusing because IAM policy documents and role assume policy document have the same syntax but different meaning.

I first thought of moving it but then I realised that we'll do it later (IF needed - YAGNI) when we'll add the logic to create the IAM role for the users to remove this future duplication.

For now, let's leave it inline and we'll move it out later when we'll need to have slightly different assume role policies for the users (because they'll need the additional statement for the SAML federated login).